gemma-core/src/main/java/ubic/gemma/persistence/service/analysis/expression/diff/ExpressionAnalysisResultSetDao.java

```diff
@@ -40,4 +44,5 @@
 
     ExpressionAnalysisResultSet thawWithoutContrasts( ExpressionAnalysisResultSet resultSet );
 
+    ExpressionAnalysisResultSetValueObject loadValueObjectsLimit( int i );
```

Comment:
This is the main way repository layers are extended with Spring Data to [support defining queries via method name](https://docs.spring.io/spring-data/jpa/docs/1.5.0.RELEASE/reference/html/repositories.html). Although `load...By` is not really a support prefix, `find...By` is already used for another purpose in BaseDao.